tests/unit/custom-step-form.spec.ts

```diff
@@ -0,0 +1,43 @@
+import CustomStepForm from '@/components/stepforms/CustomStepForm.vue';
+import { BasicStepFormTestRunner } from './utils';
+
+
+interface ValidationError {
```

Comment:
This interface is not used anymore :) This is strange that `yarn lint`doesnt detect it. I will try to fix the configuration